### ISSUE TITLE
Force a tab to start at a certain scroll position

### ISSUE THREAD

**Issue by tcorreiaubi** [PedroBern/react-native-collapsible-tab-view#98]
Hey(once again), I was trying to force a tab to always start at a certain scroll position, this is a really specific behavior, what I tried to do was on the indexChange call the scrollTo on that tab to place it on the position where I need, example
```jsx
      onIndexChange={(onChangeData) => {
            const {tabName} = onChangeData;
            if (tabName === TabKey.MAP) {
              (mapRef.current as ScrollView).scrollTo({
                y: IMAGE_SLIDER_HEIGHT - HEADER_BAR_HEIGHT,
                animated: true,
              });
            }
          }}
```

This works great for the current tab but the others tab don't sync their position properly, should I call another function or just update some value from the context? Thanks once again

**Comment by andreialecu** [comment]
This should be fixed by #93, and #99 also makes it simpler.

**Comment by tcorreiaubi** [comment]
@andreialecu Hey, nice to know, while trying to debug it I noticed that this only occurs if snap is enabled
